pkg/iter/sc_finder.go

```diff
@@ -207,12 +211,16 @@ func (m *SubclusterFinder) buildObjList(ctx context.Context, list client.ObjectL
 // shouldSkipBasedOnSandboxState returns true if the object whose labels
 // is passed does not belong to the given sandbox or main cluster
 func shouldSkipBasedOnSandboxState(l map[string]string, sandbox string) bool {
+	// Note, for a pod, the labels passed in will be from the statefulset. So,
```

Comment:
Is it better to add this to the function's description?

Reply:
yeah, that makes sense. There are 3 functions that have this comment. So, I'll change all 3.